src/adhocracy_core/adhocracy_core/sheets/badge.py

```diff
@@ -138,6 +138,7 @@ class BadgeAssignmentSchema(colander.MappingSchema):
     subject = Reference(reftype=BadgeSubjectReference)
     badge = Reference(reftype=BadgeReference)
     object = Reference(reftype=BadgeObjectReference)
+    description = Text()
```

Comment:
This isn't needed as it already comes with the `IDescription` sheet which you added to the resource above.

(Btw this was also wrong with the `IBadge` resource, but fixed with #1270).